homeassistant/components/xiaomi_miio/manifest.json

```diff
@@ -4,6 +4,6 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/xiaomi_miio",
   "requirements": ["construct==2.9.45", "python-miio==0.5.3"],
-  "codeowners": ["@rytilahti", "@syssi"],
+  "codeowners": ["@rytilahti", "@syssi", "starkillerOG"],
```

Comment:
`"@starkillerOG"`

Reply:
whoops, thanks for pointing it out!
just fixed it